File: Parciales/1erParcial/models/Libro.js
const mongoose = require('mongoose');

const LibroSchema = new mongoose.Schema({
    titulo: { type: String, required: true },
    autor: { type: String, required: true },
    editorial: { type: String, required: true },
    anio: { type: Number, required: true },
    descripcion: { type: String, required: true },
    num_paginas: { type: Number, required: true}
});

module.exports = mongoose.model('Libro', LibroSchema);

File: Practicas/LibrosGraphQL/Models/Libro.cs
using System;

namespace LibrosGraphQL.Models
{
    public class Libro
    {
        public int Id { get; set; }
        public string Titulo { get; set; } = string.Empty;
        public string Autor { get; set; } = string.Empty;
        public string Editorial { get; set; } = string.Empty;
        public int Anio { get; set; }
        public string Descripcion { get; set; } = string.Empty;
        public int NumPaginas { get; set; }
    }
}